includes/logic-apps-resource-environment-differences-table.md

```diff
@@ -9,6 +9,6 @@ ms.date: 02/23/2023
 |---------------|----------|----------------------------|---------------------------------------------------------------------------|-----------------------------------------------------------------------------|
 | **Logic App (Consumption)** <br><br>Host environment: Multi-tenant Azure Logic Apps | - Easiest to get started <br><br>- Pay-for-what-you-use <br><br>- Fully managed | A single logic app can have *only one* workflow. <br><br>Logic apps *across Azure Active Directory tenants* share the same processing (compute), storage, network, and so on. <br><br>For redundancy purposes, data is replicated in the [paired region](../articles/reliability/cross-region-replication-azure.md). For high availability, [geo-redundant storage (GRS)](../articles/storage/common/storage-redundancy.md#geo-redundant-storage) is enabled. | [Consumption](../articles/logic-apps/logic-apps-pricing.md#consumption-pricing) (pay-per-execution) | Azure Logic Apps manages the default values for these limits, but you can change some of these values, if that option exists for a specific limit. |
-| **Logic App (Consumption)** <br><br>Host environment: <br>[Integration service environment (ISE)](../articles/logic-apps/connect-virtual-network-vnet-isolated-environment-overview.md) | - Enterprise scale for large workloads <br><br>- 20+ ISE-specific connectors that connect directly to virtual networks <br><br>- Predictable pricing with included usage and customer-controlled scaling | A single logic app can have *only one* workflow. <br><br>Logic apps *in the same environment* share the same processing (compute), storage, network, and so on. <br><br>Data stays in the same region where you deploy the ISE. | [ISE](../articles/logic-apps/logic-apps-pricing.md#ise-pricing) (fixed) | Azure Logic Apps manages the default values for these limits, but you can change some of these values, if that option exists for a specific limit. |
+| **Logic App (Consumption)** <br><br>Host environment: <br>[Integration service environment (ISE)](../articles/logic-apps/connect-virtual-network-vnet-isolated-environment-overview.md) <br><br>**Note**: Since November 1, 2022, you can no longer create an ISE. On August 31, 2024, ISE will retired.| - Enterprise scale for large workloads <br><br>- 20+ ISE-specific connectors that connect directly to virtual networks <br><br>- Predictable pricing with included usage and customer-controlled scaling | A single logic app can have *only one* workflow. <br><br>Logic apps *in the same environment* share the same processing (compute), storage, network, and so on. <br><br>Data stays in the same region where you deploy the ISE. | [ISE](../articles/logic-apps/logic-apps-pricing.md#ise-pricing) (fixed) | Azure Logic Apps manages the default values for these limits, but you can change some of these values, if that option exists for a specific limit. |
```

Comment:
Hi @mbsnl, 

Thanks for your contribution! Please update with the following wording:

```suggestion
| **Logic App (Consumption)** <br><br>Host environment: <br>[Integration service environment (ISE)](../articles/logic-apps/connect-virtual-network-vnet-isolated-environment-overview.md) <br><br>**Note**: On August 31, 2024, the ISE option will retire. Since November 1, 2022, you can no longer create an ISE. Instead, you can create a Standard logic app, which runs in single-tenant Azure Logic Apps, can include multiple workflows, and provides the same capabilities as an ISE plus more. | - Enterprise scale for large workloads <br><br>- 20+ ISE-specific connectors that connect directly to virtual networks <br><br>- Predictable pricing with included usage and customer-controlled scaling | A single logic app can have *only one* workflow. <br><br>Logic apps *in the same environment* share the same processing (compute), storage, network, and so on. <br><br>Data stays in the same region where you deploy the ISE. | [ISE](../articles/logic-apps/logic-apps-pricing.md#ise-pricing) (fixed) | Azure Logic Apps manages the default values for these limits, but you can change some of these values, if that option exists for a specific limit. |
```